sdk/swagger.yaml

```diff
@@ -177,6 +177,74 @@ components:
         schema: {}
         executionContext:
           $ref: "#/components/schemas/EvaluatedExecutionContext"
+    EnvironmentTag:
+      type: object
+      properties:
+        id:

```

Comment:
we dropped this from the api response so I think we can ignore it here too

Reply:
fixed